source/partials/next_release/_release.md.erb

```diff
@@ -1,51 +1,19 @@
-<h4>Security Fixes</h4>
+<h4>Bundled Plugins</h4>
 
-This release has critical security fixes to prevent CSRF, XSS and Session hijacking exploits. We highly recommend users to upgrade to keep your GoCD server and agent secure.
-Thanks [4cad](https://hackerone.com/4cad) for reporting these issues. 
+With this release GoCD provides support for [Pipleine as code](https://docs.gocd.org/current/advanced_usage/pipelines_as_code.html) through [JSON config repo plugin](https://github.com/tomzo/gocd-json-config-plugin) and the [YAML config repo plugin](https://github.com/tomzo/gocd-yaml-config-plugin). These plugins maintained by our core contributor [Tomasz Sętkowski](https://github.com/tomzo) will be bundled by default.
 
 <h4>Improvements</h4>
 
-* <%= link_to_commit '42c681d1585db99747877830c2b1fdc2ca3e80b1', 'Added option to skip agent startup on windows.', 3577 %>
+* <%= link_to_commit '1ccb0294265b4baea091c9068d7813fdde5009b4', 'Ensure order of auth_config in config xml is used to derive the order in which plugins are used for authentication.', 3608 %>
+* <%= link_to_commit '36d546e680aa43c31bf29d2a586ca31e57739bde', 'Remove tfs-impl-10' %>
+* <%= link_to_commit 'd7c4d167fc10101b03af4cd6676c806cef244747', 'Paused pipelines shows the pause reason on the history page.', 2784 %>
+* <%= link_to_commit '52ac924aad78ad2795c088199eb4c70f14b8205a', 'Save agent hostname and ip address as part of build history. This information will be displayed on the job details page for all kinds of agents.', 3368 %>
```

Comment:
Removed support for Team Foundation Server 2010. Users can continue to use GoCD with TFS 2012, 2013, 2015 and Visual Studio Team Services.